hkm/utils.py

```diff
@@ -0,0 +1,16 @@
+def encoded_dict(in_dict):
+    def handle_encoding(v):
+        if isinstance(v, unicode):
+            return v.encode('utf8')
+        elif isinstance(v, str):
+            # Must be encoded in UTF-8
+            return v.decode('utf8')
+        elif isinstance(v, list):
+            # Recursive call
+            return [handle_encoding(vv) for vv in v]
+        return v
```

Comment:
```suggestion
    from django.utils.encoding import force_text
    
    def handle_encoding(v):
        if isinstance(v, list):
            # Recursive call
            return [handle_encoding(vv) for vv in v]
        return force_text(v, strings_only=True)
```

Reply:
I could not find any utils for this, but I will test this one tomorrow.

Reply:
I tried it already: 
![image](https://user-images.githubusercontent.com/389204/144062808-9538df10-5839-4167-81ca-c60ba0845256.png)

I can try more tomorrow, but don't want to waste much time with this.